goserver/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/google/meet-on-fhir/server"
+)
+
+var authorizedFhirURL = flag.String("authorized_fhir_url", "", "The FHIR base url that is authorized to launch the telehealth app. If not set, launch endpoint will always return HTTP 401.")
```

Comment:
Update message about what will happen if this isn't set.

Reply:
Done.